.github/workflows/build-frontend.yaml

```
@@ -0,0 +1,28 @@
+    name: Build React App
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)